storybook/main.ts

```diff
@@ -23,7 +23,7 @@ const getStoryPathsFromEnv = (): string[] | false => {
 const defaultStoryPaths = [
   "../packages/**/*.stories.tsx",
   "../packages/**/*.stories.mdx",
-  "../draft-packages/**/*.stories.tsx",
+  "../draft-packages/**/!(deprecated.)*.stories.tsx",
```

Comment:
This introduces a new pattern where we exclude stories that are prefixed with `deprecated.` from being loaded into storybook